### PR TITLE
Delete obsolete strings in de (DEV)

### PR DESCRIPTION
**Pull request by JohannesZeilinger**
- Deleted obsolete strings without usages
- Corrected some spelling mistakes
- Minor changes

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -1634,11 +1500,11 @@
     <!-- XTXT: Statistics number of fully vaccinated people title -->
     <string name="statistics_fully_vaccinated_people_title">"Vollständig geimpfte Personen"</string>
     <!-- XTXT: Statistics number of fully vaccinated people text -->
-    <string name="statistics_fully_vaccinated_people_text">"Anzahl der an das RKI übermittelten Personen, die die alle notwendigen Impfdosen erhalten haben."</string>
+    <string name="statistics_fully_vaccinated_people_text">"Anzahl der an das RKI übermittelten Personen, die alle notwendigen Impfdosen erhalten haben."</string>
     <!-- XTXT: Statistics number of vaccinated people at least title -->
     <string name="statistics_people_vaccinated_at_least_once_title">"Mindestens einmal geimpfte Personen"</string>
     <!-- XTXT: Statistics number of vaccinated people at least text -->
-    <string name="statistics_people_vaccinated_at_least_once_text">"Anzahl der an das RKI übermittelten Personen, die die erste Impfdosis erhalten haben."</string>
+    <string name="statistics_people_vaccinated_at_least_once_text">"Anzahl der an das RKI übermittelten Personen, die erste Impfdosis erhalten haben."</string>
```
Comment (InkenTownsend) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1507]:
```suggestion
    <string name="statistics_people_vaccinated_at_least_once_text">"Anzahl der an das RKI übermittelten Personen, die die erste Impfdosis erhalten haben."</string>
```

Comment (InkenTownsend) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:1507]: Here it needs to stay.

**Comment by InkenTownsend** [comment]
@JohannesZeilinger There was one suggested change that needs to be reverted. See my comment. Thank you!

**Comment by JohannesZeilinger** [comment]
@InkenTownsend Thank you for your suggestion. I accepted your change.

**Review by mtwalli — APPROVED** [review]
Lgtm, tested the release build and seems fine.

**Review by InkenTownsend — APPROVED** [review]
UA approved

**Pull request merged**